caluma/caluma_logging/middleware.py

```diff
@@ -0,0 +1,22 @@
+import json
+
+from caluma.caluma_logging.models import AccessLog
+
+
+class AccessLogMiddleware:
+    def __init__(self, get_response):
+        self.get_response = get_response
+
+    def __call__(self, request):
+        response = self.get_response(request)
+        # user is available only after the request was processed
+        body = json.loads(request.body.decode("utf-8"))
+
+        AccessLog.objects.create(
+            username=request.user.username,
+            query=body.get("query"),
+            variables=body.get("variables"),
+            operation_name=body.get("operationName"),
```

Comment:
It'd make sense to also write the status code of the response into the DB, so we can distinguish between successful and unsuccessful requests.

Reply:
I thought we always return 200, even if there are errors...?!

Reply:
We also return 401. And yes, for the 200s we still don't know if they produced some error (could be tracked too). But at least we could filter the 401s.

Reply:
Included that status code